src/tests/unit/tests/DetailsView/reports/components/report-sections/summary-section.test.tsx

```diff
@@ -2,19 +2,21 @@
 // Licensed under the MIT License.
 import { shallow } from 'enzyme';
 import * as React from 'react';
-
+import { DateProvider } from '../../../../../../../common/date-provider';
 import { SectionProps } from '../../../../../../../DetailsView/reports/components/report-sections/report-section-factory';
 import { SummarySection } from '../../../../../../../DetailsView/reports/components/report-sections/summary-section';
 
 describe('SummarySection', () => {
     const pageTitle = 'page-title';
     const pageUrl = 'url:target-page';
+    const scanDate = new Date('2019-05-29T19:12:16.804Z');
+    const getUTCStringFromDateStub: typeof DateProvider.getUTCStringFromDate = anyDate => '2018-03-12 11:24 PM UTC';
 
     const props: SectionProps = {
```

Comment:
you're adding the `utcDateConverter` here because of the type we're using (`SectionProps`) but actually, we don't need to use that type, `SummarySectionProps` is what we need. Wanna update?